apps/FeedReader.md

```diff
@@ -19,24 +19,22 @@ links:
     url: "https://jangernert.github.io/FeedReader/"
   - type: GitHub
     url: jangernert/FeedReader
-  - type: Launchpad
-    url: feedreader
   - type: Translations
     url : "https://www.transifex.com/dev-feedreader/feedreader/"
   - type: License
-    url: "https://bazaar.launchpad.net/~eviltwin1/feedreader/trunk-1/view/head:/COPYING"
+    url: "https://github.com/jangernert/FeedReader/blob/master/COPYING"
 
 screenshots:
   - FeedReader.png
 
 installation:
   - system: elementary
     info:
-    - version: 0.3
+    - version: Stable
```

Comment:
`version` currently refers to the version number of the OS / elementary (so 0.3 was for Freya, 0.4 for Loki). Whether a PPA is "stable" or for "development" is determined by the `type`. Am I right that FeedReader currently support Freya *and* Loki?